crud/common/utils.lua

```diff
@@ -1177,6 +1194,61 @@ function utils.storage_info_on_storage()
     return {status = "running"}
 end
 
+--- Initializes a storage function by its name.
+--
+--  It adds the function into the global scope by its name and required
+--  rights to a vshard storage user.
+--
+--  @function init_storage_call
+--
+--  @param string name a name of the function.
+--  @param function func the function.
+--
+--  @return nil
+function utils.init_storage_call(name, func)
+    dev_checks('string', 'function')
+
+    rawset(_G['_crud'], name, func)
+    name = utils.get_storage_call(name)
+
+    if type(box.cfg) ~= 'table' or box.cfg.read_only then
+        -- If the storage is not configured yet then it could be
+        -- a unit-test or something else.
+        --
+        -- Anyway, it is out of scope of the call to detect and to raise an
+        -- error for the case.
+        return
+    end
+
+    local ok, storage_info = pcall(vshard.storage.info)
+    if not ok then
+        -- The storage is not configured. This is also none of our business.
```

Comment:
I think it actually is

Reply:
The check (+error) is moved to `crud.init_storage()` call.